experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -1431,11 +1431,65 @@ function dim_of_simple_module(L::LieAlgebra, hw::Vector{<:IntegerUnion})
 end
 
 @doc raw"""
-    dominant_character(L::LieAlgebra{C}, hw::Vector{Int}) -> Dict{Vector{Int}, Int}
+    dominant_weights([T,] L::LieAlgebra{C}, hw::Vector{<:IntegerUnion}) -> Vector{T}
```

Comment:
```suggestion
    dominant_weights([T = Vector{Int}], L::LieAlgebra{C}, hw::Vector{<:IntegerUnion}) -> Vector{T}
```

Reply:
I did not want to specify this at this point, as I want to straight out all of the return types in the near future. That's also the reason for the many `The return type will change in the future" in docstrings.